lib/trento/application/integration/discovery/payloads/cluster/cluster_discovery_payload.ex

```diff
@@ -77,10 +78,28 @@ defmodule Trento.Integration.Discovery.ClusterDiscoveryPayload do
     do_detect_cluster_type(has_sap_hana_topology, has_sap_hana, has_sap_hana_controller)
   end
 
+  defp parse_cluster_type(%{"crmmon" => %{"groups" => groups}}) do
```

Comment:
This clause matches if the above one is not `nil`.  Maybe it is clearer if we invert the order and add `"clones" => nil` to this one, wdyt?

Reply:
Right! It looks even better. Thank you!
Changed!